neo/app/views/index.scala.html

```diff
@@ -1,15 +1,31 @@
 @(title: String) @main(title){
 <div xmlns="http://www.w3.org/1999/html" ng-controller="AppCtrl">
 
-  <map lat="39.5" lng="-96.35" zoom="4"></map>
+  <div class="Map">
 
-  <search-bar></search-bar>
 
-  <time-series id="chart"></time-series>
+    <map lat="39.5" lng="-96.35" zoom="4"></map>
 
+    <div id="logo">
+      <img src ="https://asterixdb.apache.org/img/asterixdb_tm.png" id="logo_img">
+    </div>
+
+    <div id='how-to'>
+      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info">How to use this demo</a>
+    </div>
+
+  </div>
+
+  <div class="SearchBar">
+    <search-bar></search-bar>
+  </div>
+
+  <div class="TimeSeries">
```

Comment:
let's unify the js class name pattern. I think it's more common to use all-lower-case with `-` in js.